### ISSUE TITLE
Updation needed

### ISSUE THREAD

**Issue by Ayushk4** [metakgp/metakgp.github.io#3]
- New projects need to be added
- Some touch by @xypnox might be good.

**Comment by sahil-shubham** [comment]
Hey @Ayushk4, Can you add more details regarding this issue? or break it down into few other issues so that I could also contribute.

**Comment by Ayushk4** [comment]
It's great that you are interested in contributing. 

There have been a lot of new projects by metakgp since since was updated - Chillzone, Gyft, wimp, kronos, iqps. Maybe we could add those too. 

> Some touch by @xypnox might be good.
Maybe some css styling or updates to the webpage could be made.

I don't have any concrete issue in mind currently but seems like there is a lot of room for work.

Ping @thealphadollar what do you say.

**Comment by kaustubhhiware** [comment]
This is a great idea @Ayushk4 !

For the projects mentioned, we can add one line descriptions for each of the mentioned projects in [params.json](https://github.com/metakgp/metakgp.github.io/blob/master/params.json).

Additionally, how about making the cards searchable somehow ? Might be an addition to the issue but would be great.

Additional reference: https://wiki.metakgp.org/w/Metakgp:MetaX

**Comment by sahil-shubham** [comment]
@kaustubhhiware That idea of making the cards searchable is great! 

How about we also add the list of metakgp's founding members and the current maintainers? As this is a website even the users who don't use github, or who don't even code may open, we need to add something for them too. 

We need a proper README too.

cc/ @thealphadollar

**Comment by thealphadollar** [comment]
@sahil-shubham Yeah! It would be cool to have projects searchable.

> How about we also add the list of metakgp's founding members and the current maintainers? 

This idea is really amazing. We have a sheet (check description of #meta-x) which lists all the metakgp projects, their tech-stack, their maintainers, etc. It might be helpful for collecting data.

Another cool feature we can add is that we feature the top three contributors of the month to the metakgp organization repos. Do you think this would be good to have?

**Comment by sahil-shubham** [comment]
> It might be helpful for collecting data.

Yeah, I checked that sheet and it is exactly what we needed, thanks a lot. We would link the github profiles of the these maintainers on the website. 

> Another cool feature we can add is that we feature the top three contributors of the month to the metakgp organization repos

@thealphadollar I think it's a great way of promoting the beginners who have just started with open source. I did look up and saw that we used to share the weekly top contributors on metakgp's facebook page in 2016-17, but a month is a better time period for this.
